es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md

```diff
@@ -0,0 +1,509 @@
+          "master_fingerprint": "d6043800",
+          "path": "m/0'/0'/18'"
+        }
+      ]
+    }
+  ],
+  "outputs": [
+    {
+      "bip32_derivs": [
+        {
+          "pubkey": "029bb586a52657dd98852cecef78552a4e21d081a7a30e4008ce9b419840d4deac",
+          "master_fingerprint": "d6043800",
+          "path": "m/0'/1'/27'"
+        }
+      ]
+    },
+    {
+    }
+  ],
+  "fee": 0.00028800
+}
+```
+Como puede ver, creó el PSBT y luego lo actualizó con toda la información que pudo encontrar localmente.
+
+A partir de ahí, debe usar `walletprocesspsbt` para Finalizar, como de costumbre:
+```
+$ psbt_new_f=$(bitcoin-cli walletprocesspsbt $psbt_new | jq -r '.psbt')
+```
+Luego, un análisis mostrará que tambien está a punto de estar lista para ser enviada:
+```
+$ bitcoin-cli analyzepsbt $psbt_new_f
+{
+  "inputs": [
+    {
+      "has_utxo": true,
+      "is_final": true,
+      "next": "extractor"
+    },
+    {
+      "has_utxo": true,
+      "is_final": true,
+      "next": "extractor"
+    }
+  ],
+  "estimated_vsize": 288,
+  "estimated_feerate": 0.00100000,
+  "fee": 0.00028800,
+  "next": "extractor"
+}
+```
+Ahora, ¿realmente querría usar `walletcreatefundedpsbt` si estuviera creando un programa `bitcoin-cli`? Probablemente no. Pero es el mismo análisis que si se debe usar `fundrawtransaction`. ¿Deja que Bitcoin Core haga el análisis, el cálculo y las decisiones, o lo toma usted mismo?
+
+## Envíe una PSBT
+#### Rol en la PSBT: Extractor
+
+Para finalizar el PSBT, use `finalizepsbt`, que convertirá su PSBT nuevamente en hexadecimal. (También asumirá el rol de finalizador, si eso no sucedió ya).
+```
+$ bitcoin-cli finalizepsbt $psbt_f
+{
+  "hex": "020000000001021b95a6055174ec64b82ef05b6aefc38f34d0e57197e40281ecd8287b4260dec60100000000ffffffff1cf4c7f70160c9fe22436c70e032033fcb7d22309e3d60e386a82cf1f5ef48870000000000ffffffff018a02000000000000160014c772d6f95542e11ef11e8efc7c7a69830ad38a050247304402200a896f1cfa741ccda5f683bb85d95dbf836c390cdd45a785ece1f54766795821022051c4c956da8d3012da61b5465158888939a1ad5dfb106c1e25a2f92f13879122012102bac3a645fd7a0a537c627ad37f5f695058c58d19ceb0033e913f1a9975e00dee0247304402207eb0d3423b8c62150f83a0756baaa93dc8cd15e07f594b2131c753d6ed16e3b2022051e5a44372fb680bf5c487bb87ef0f65bf040ac5e0d54cdbb4f5bcc1ddaac745012102883bb5463e37d55252d8b3d5c2141b007b37c8a7db6211f75c955acc5ea325eb00000000",
+  "complete": true
+}
+```
+Como de costumbre, querrá guardar eso y luego puede enviarlo
+```
+$ psbt_hex=$(bitcoin-cli finalizepsbt $psbt_f | jq -r '.hex')
+$ bitcoin-cli -named sendrawtransaction hexstring=$psbt_hex
+ea73a631b456d2b041ed73bf5767946408c6ff067716929a68ecda2e3e4de6d3
+```
+## Revise el flujo de trabajo
+
+Al crear el software `bitcoin-cli`, lo más probable es que cumpla las cinco funciones principales de los PSBT con `createpsbt`, `walletprocesspsbt` y `finalizepsbt`. Esto es lo que parece ese flujo:
+
+![](images/psbt-roles-for-cli-1.png)
```

Comment:
- `![](images/psbt-roles-for-cli-1.png)` should probably be `![](../images/psbt-roles-for-cli-1.png).`